draft-meynell-panrg-scion-research-questions.md

```diff
@@ -470,12 +472,19 @@ Is reverse path refresh a relevant problem?
 * What time synchronization precision should we expect at the border router level of every AS? How far can we go realistically?
 -->
 
-# Interfaces for Path Awareness
+# Interfacing SCION with existing technologies
+The questions posed here are:
 
-* IPv6 in the Data Plane
-* SCION-IP translation
+* Are there any existing protocols/solutions that could be adapted to carry SCION traffic?
+* If there are not suitable out-of-the-box, what would be required to make them work?
```

Comment:
Sorry, Github refuses to do replacement suggestion because there is a deleted line in there...


```
# Interfacing SCION with existing technologies
The questions posed are about adapting SCION to be interoperable or compatible with existing technologies and protocols:
* For which existing technologies/protocols do we want compatibility/interoperability?
* How would we need to adapt SCION?
* How much work is it?
```

Reply:
@knmeynell Can you PTAL at this section and write it in proper english?
We tried to be clear, but we both don't agree on the exact words to use here.
The idea is explained in the text of this section. Please feel free to ask us about what we really want to express here :smile: (maybe we don't exactly know what we want :wink: )

Reply:
JFYI Kevin is off this week, he might not respond